safe_transaction_service/tokens/tasks.py

```diff
@@ -75,29 +94,21 @@ def update_token_info_from_token_list_task() -> int:
         except TokenListRetrievalException:
             logger.error("Cannot read tokens from %s", token_list)
 
-    if not tokens:
-        return 0
-
-    # Make sure current chainId matches the one in the list
     current_chain_id = get_ethereum_network().value
-    ethereum_client = get_auto_ethereum_client()
-
-    token_addresses = []
-    for token in tokens:
-        if token.get("chainId") == current_chain_id:
-            token_address = token["address"]
-            if token_address.startswith("0x"):
-                token_addresses.append(fast_to_checksum_address(token_address))
-            else:
-                # Try ENS resolve
-                try:
-                    if resolved_address := ethereum_client.w3.ens.address(
-                        token_address
-                    ):
-                        token_addresses.append(resolved_address)
-                except (ValueError, Web3Exception):
-                    logger.warning("Cannot resolve %s ENS address", token_address)
 
+    # Some lists are meant to be used for multiple chains
+    filtered_tokens = [
+        token for token in tokens if token.get("chainId") == current_chain_id
+    ]
+    if not filtered_tokens:
+        return 0
+
+    tokens_updated_count = 0
     with transaction.atomic():
         Token.objects.update(trusted=False)
-        return Token.objects.filter(address__in=token_addresses).update(trusted=True)
+        for token in filtered_tokens:
+            token_address = _parse_token_address_from_token_list(token["address"])
+            tokens_updated_count += Token.objects.filter(address=token_address).update(
+                logo_uri=token.get("logoURI") or "", trusted=True
```

Comment:
```suggestion
                logo_uri=token.get("logoURI", ""), trusted=True
```

Reply:
This will not work as expected. If in the JSON `logoUri` holds `null`, we will try to store a `None` instead of `empty string`